### ISSUE TITLE
[BUG]: News overview page has no max width yet

### ISSUE THREAD

**Issue by irisvw** [fdnd-agency/nebulaxplorer#374]
### Description

The News overview page has no max width yet.

### Expected behavior

It should have a max width so that it's still easily legibile at larger screen widths.

### Actual behavior

The page gets so wide that the content is hard to read.

### What device are you experiencing the problem on?

Desktop

### What operating system are you experiencing the problem on?

Windows 11

### What browsers are you experiencing the problem on?

Chrome
